docs/UserGuide.md

```diff
@@ -291,16 +298,22 @@ _Want to delete your account? Call this command to delete your account and clear
 
 > Adds a customer to the address book.
 
-_Did you just receive a new customer? You can add the customer's details into HomeBoss with this command._
+_You can add a new customer's details into HomeBoss with this command._
 
-**Format:** `customer add --name NAME --phone PHONE_NUMBER --email EMAIL --address ADDRESS`
+**Format:** `customer add --name NAME --phone PHONE_NUMBER --email EMAIL --address ADDRESS`{.swift}
 
-<box background-color="#dff0d8" border-color="#d6e9c6" type="info" header="Note">
-    * All fields are required.
-    * HomeBoss does not allow you to add customers with the same phone number.
+<box type="note" background-color="#dff0d8" border-color="#d6e9c6" icon=":information_source:">
+
+**Note:**
+* All fields are required.
+* HomeBoss does not allow you to add customers with the same phone number.
+* A unique customer ID will be assigned to each customer.
```

Comment:
Would it be good to specify parameter constraints, e.g. phone is 8 digit. And for the other commands as well

Reply:
Good catch, have added in the parameter constraints for all parameters